packages/datetime/src/components/react-day-picker/datePicker3Caption.tsx

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright 2023 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import classNames from "classnames";
+import * as React from "react";
+import { CaptionLabel, type CaptionProps, useDayPicker, useNavigation } from "react-day-picker-8";
+import innerText from "react-innertext";
+
+import { Button, DISPLAYNAME_PREFIX, HTMLSelect, type OptionProps } from "@blueprintjs/core";
+import { ChevronLeft, ChevronRight } from "@blueprintjs/icons";
+
+import { DateUtils, Months } from "../../common";
+import { DatePicker3CaptionClasses as CaptionClasses, ReactDayPickerClasses } from "../../common/classes";
+import { useMonthSelectRightOffset } from "../../common/useMonthSelectRightOffset";
+import { DatePicker3Context } from "../date-picker3/datePicker3Context";
```

Comment:
Imports changed when ported over to new package. All other code remains the same.
https://github.com/palantir/blueprint/blob/ca86eebdf9e7ed242d0cfd8abaf73ac376c9a5f1/packages/datetime2/src/components/react-day-picker/datePicker3Caption.tsx#L22-L28